libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -544,14 +544,20 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
     }
 
     @ReactMethod
-    fun fetchReverseSwapFees(promise: Promise) {
+    fun fetchReverseSwapFees(reqData: ReadableMap, promise: Promise) {
         executor.execute {
-            try {
-                val reverseSwapFees = getBreezServices().fetchReverseSwapFees()
-                promise.resolve(readableMapOf(reverseSwapFees))
-            } catch (e: SdkException) {
-                e.printStackTrace()
-                promise.reject(e.javaClass.simpleName, e.message, e)
+            val reverseSwapFeesRequest = asReverseSwapFeesRequest(reqData)
+
+            if (reverseSwapFeesRequest == null) {
```

Comment:
asReverseSwapFeesRequest will always return a ReverseSwapFeesRequest

Reply:
Done in b353354db83fbba408645f2007ca2172cbfeba2c